packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-pwd-recovery-mfa/java/summaryofsteps.md

```diff
@@ -0,0 +1,9 @@
+## Summary of steps
+
+The following are the sequence of steps for the password recovery use case.
+
+<div class="common-image-format">
+
+![Displays the sequence of steps for the Java SDK password recovery flow](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-java.png)
```

Comment:
This is mostly looking really good; I just had a couple of consistency comments:

1. "Checks email for the verification code" — change "Checks" to "Check" for consistency with other user steps (it's that commands versus actions thing again)
2. For the first method call (`IDXAuthenticationWrapper.begin()`) You've included the name of the class plus the parens after the method. Since space is an issue in these diagrams (and for consistency), it would be better not to include the class name, but I would like to see the parens included after all the method names.

Reply:
1. Opps! I missed "Checks". Thanks!
2. I will add parens in the other method. But for the first method call, I think it is good to add the full class name to indicate that we are using the IDXAuthenticationWrapper class? `begin()` doesn't show enough info? Thoughts? This step was recently added by the SME. I will need to update all my previous diagrams to update this info.